docs/docs/secrets-managers.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: secrets-managers
+sidebar_label: Secrets Managers
+title: Secrets Managers
+description: Credentials used to authenticate to external services are safely stored in a secrets manager.
+abstract: Credentials managers are used to safeguard credentials used to authenticate to external services.
+---
+
+<!-- this file is version specific, do not use `@site/...` syntax -->
+import RasaProLabel from '@theme/RasaProLabel';
+import RasaProBanner from '@theme/RasaProBanner';
+
+<RasaProLabel />
+<RasaProBanner/>
+
+## Secrets Managers
+Secrets managers are used to safeguard credentials used to authenticate to external services,
+like databases.
+
+Rasa Pro supports the following secrets managers:
+* [HashiCorp Vault](https://www.hashicorp.com/products/vault)
+
+
+Currently, Rasa Pro supports sefeguarding credentials for the following services:
+- [Tracker Stores](./tracker-stores.mdx)
+
+
+## Vault Secrets Manager
+
+The Vault Secrets Manager is used to store credentials used to authenticate to external services.
+The credentials are stored in a Vault instance and can be encrypted at rest.
```

Comment:
We should rephrase to active voice and provide a link to Vault documentation/tutorial on how to store credentials in a Vault instance, e.g. `To store credentials in a Vault instance, you can read the official Vault docs. You can also encrypt credentials at rest.`

Reply:
Perfect 👍🏻